@navikt/core/react/src/form/combobox/ComboboxProvider.tsx

```diff
@@ -58,6 +58,12 @@ const ComboboxProvider = forwardRef<HTMLInputElement, ComboboxProps>(
     const options = mapToComboboxOptionArray(externalOptions) || [];
     const filteredOptions = mapToComboboxOptionArray(externalFilteredOptions);
     const selectedOptions = mapToComboboxOptionArray(externalSelectedOptions);
+
+    const [isFirefoxOnAndroid, setIsFirefoxOnAndroid] = useState(false);
+    useEffect(() => {
+      setIsFirefoxOnAndroid(/Android.+Firefox\//.test(navigator.userAgent));
+    }, []);
```

Comment:
Do we need to use `useState` and `useEffect` for this?

Reply:
Can test without in `/examples/next-appdir` if we want to make sure.

Reply:
Maybe something like this could work:
```ts
const isFirefoxOnAndroid = /Android.+Firefox\//.test(
  typeof navigator === "undefined" ? "" : navigator.userAgent,
);
```

Reply:
The above code seems to be working in Nextjs. If we care about micro-optimizations we can use `includes()` instead, since regex is known to be a bit slow in JS. I guess it would be something like this:
```ts
    const userAgent =
      typeof navigator === "undefined" ? "" : navigator.userAgent;
    const isFirefoxOnAndroid =
      userAgent.includes("Android") && userAgent.includes("Firefox/");
```

Reply:
☝️ Can confirm works with nextjs app-router, astro and shadow-dom

Reply:
![image](https://github.com/user-attachments/assets/46de09a4-5910-4d12-bcad-128e30261880)
indexOf og Includes er best confirmed! (med en veldig basic test da 😂 )
https://www.measurethat.net/Benchmarks/Show/14772/0/includes-vs-test-vs-match-vs-indexof

Reply:
Can't see us having any performance boosts by memoizing it, so should be fine 🤞

Reply:
All right! If someone could re-approve this PR, then I'll merge it.